regress/core/out_marc21.sql

```diff
@@ -0,0 +1,888 @@
+--
+-- AsMARC21 regression test
+-- Copyright (C) 2021 University of Münster (WWU), Germany
+-- Written by Jim Jones <jim.jones@uni-muenster.de>
+
+CREATE TABLE asmarc21_rt (gid int, geom geometry);
+INSERT INTO asmarc21_rt (gid,geom) VALUES
+	(1,'SRID=4326;GEOMETRYCOLLECTION(
```

Comment:
To slim things down, can you run your test geometries through ST_AsText(geom, 5)?